coeus-impl/src/main/java/org/kuali/coeus/common/framework/custom/attr/CustomAttributeDocument.java

```diff
@@ -24,7 +24,7 @@
 import org.kuali.rice.kew.doctype.bo.DocumentType;
 
 /**
- * 
+ *
  * This class bo of CustomAttributeDocument.
  */
 public class CustomAttributeDocument extends KcPersistableBusinessObjectBase implements MutableInactivatable, CustomAttributeDocumentContract {
```

Comment:
Hi Travis,

The changes have been pushed to RESSUP-1356.

Thanks,
Chao

From: Travis Schneeberger <notifications@github.com<mailto:notifications@github.com>>
Reply-To: kuali/kc <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, March 14, 2016 1:24 PM
To: kuali/kc <kc@noreply.github.com<mailto:kc@noreply.github.com>>
Cc: Chao Yue <chaoyue@iu.edu<mailto:chaoyue@iu.edu>>
Subject: Re: [kc] [RESSUP-1356] Add Sort ID to Custom Attribute Document (#1605)


In coeus-impl/src/main/java/org/kuali/coeus/common/framework/custom/attr/CustomAttributeDocument.java<https://github.com/kuali/kc/pull/1605#discussion_r56039542>:

> @@ -24,7 +24,7 @@
>  import org.kuali.rice.kew.doctype.bo.DocumentType;
>
>  /**
> - *
> + *
>   * This class bo of CustomAttributeDocument.
>   */
>  public class CustomAttributeDocument extends KcPersistableBusinessObjectBase implements MutableInactivatable, CustomAttributeDocumentContract {


should extend KraSortablePersistableBusinessObjectBase

-
Reply to this email directly or view it on GitHub<https://github.com/kuali/kc/pull/1605/files#r56039542>.